api-compact.yaml

```diff
@@ -141,6 +142,136 @@ endpoints:
             integration.request.header.Authorization: "method.request.header.Authorization"
             integration.request.path.policy_uuid: "method.request.path.policy_uuid"
           uri: "https://${stageVariables.enterprise_access_host}/api/v1/policy-allocation/{policy_uuid}/allocate/"
+    # api/v1/assignment-configurations/{assignment_configuration_uuid}/admin/assignments/
+    learnerContentAssignmentListRequest:
+      get:
+        description: "Lists LearnerContentAssignment records, filtered by the given query parameters."
+        operationId: "api_v1_assignment_configurations_admin_assignments_list"
+        consumes:
+          - "application/json"
+        produces:
+          - "application/json"
+        parameters:
+          - *auth_header
+          - *assignment_configuration_uuid
+          - in: query
+            name: content_key
+            type: string
+            description: "The globally unique content identifier to assign to the learner.
+              Joinable with ContentMetadata.content_key in enterprise-catalog."
+            required: false
+          - in: query
+            name: content_key__in
+            type: array
+            description: "The globally unique content identifier to assign to the learner.
+              Joinable with ContentMetadata.content_key in enterprise-catalog."
+            required: false
+            items:
+              type: string
+              description: "content key"
```

Comment:
what's the difference b/w these two parameters? Also, description is same.

Reply:
There isn’t much difference between these two, first one takes a single value while the second one takes an array of `content_key` values